README.md

```diff
@@ -1,4 +1,89 @@
-# WAYF Cloud Javascript Client
+# WAYF Cloud node.js Client
```

Comment:
If this is a node specific client, I would have it in a node specific repository rather than one named "wayf-client-js". Perhaps wayf-client-node